examples/avr/display/dogm128/benchmark/main.cpp

```diff
@@ -161,7 +161,7 @@ main()
 		while (!timer.execute())
 		{
 			// rectangle in left side of screen
-			display.setColor(Color::black());
+			// TODO display.setColor(Color::black());
```

Comment:
Why is this commented out?

Reply:
setColor isn't implemented anymore for MonochromeGraphicDisplay since a monochrome display has no color. Please check the opening message of this PR.

Reply:
Ok, shouldn't it be removed then?

Reply:
It will be reimplemented